src/nspawn/nspawn-network.c

```diff
@@ -492,7 +484,7 @@ int test_network_interface_initialized(const char *name) {
 
         /* udev should be around. */
 
-        ifi = parse_interface(name);
+        ifi = resolve_ifname(NULL, name);
```

Comment:
`resolve_ifname_or_warn()`

Reply:
Good catch, but I think the old code was doing double logging. `join_bridge` is only called from `setup_bridge`, and that prints the error. We'll get something like ""Failed to add interface … to bridge …: no such device", which I think is appropriate.